easybuild/easyconfigs/i/INTEGRATE/INTEGRATE-0.2.6-foss-2018b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'INTEGRATE'
+version = '0.2.6'
+
+homepage = "https://sourceforge.net/p/integrate-fusion/wiki/Home/"
+description = """\
+ INTEGRATE is a tool calling gene fusions with exact fusion junctions and genomic breakpoints by combining RNA-Seq
+ and WGS data. It is highly sensitive and accurate by applying a fast split-read mapping algorithm based on
+ Burrow-Wheeler transform. """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
Does this software need any of the libraries in `foss` or could it be built with just the `GCC` toolchain?